app/src/main/java/projetaobcc20172/com/projetopetemfoco/activity/CadastroUsuarioActivity.java

```diff
@@ -101,24 +96,16 @@ private void cadastrarUsuario() {
             ).addOnCompleteListener(CadastroUsuarioActivity.this, new OnCompleteListener<AuthResult>() {
                 @Override
                 public void onComplete(@NonNull Task<AuthResult> task) {
-
                     if (task.isSuccessful()) {
-
-
                         String identificadorUsuario = Base64Custom.codificarBase64(usuario.getEmail());
                         usuario.setId(identificadorUsuario);
-                        usuario.salvar();
```

Comment:
Ok, o método que salva o usuário no banco não é mais chamado na tela com os dados básicos do usuário, o mesmo foi movido para quando for inserido também os dados do endereço.